source/libraries/NetworkConfigurations.ts

```diff
@@ -0,0 +1,46 @@
+import BN = require('bn.js');
+
+export type NetworkConfiguration = {
+    host: string;
+    port: number;
+    privateKey: string | undefined;
+    gasPrice: BN;
+}
+
+export type NetworkConfigurations = {
+    [networkName: string]: NetworkConfiguration;
+}
+
+export const networkConfigurations: NetworkConfigurations = {
+    ropsten: {
+        host: "ropsten.augur.net",
+        port: 8545,
+        privateKey: process.env.ROPSTEN_PRIVATE_KEY,
+        gasPrice: new BN(20)
+    },
+    kovan: {
+        host: "kovan.augur.net",
+        port: 8545,
+        privateKey: process.env.ROPSTEN_PRIVATE_KEY,
+        gasPrice: new BN(1)
+    },
+    rinkeby: {
+        host: "rinkeby.ethereum.origin.augur.net",
+        port: 8545,
```

Comment:
I believe this should be port `80`.  8545 might still work, but HAP is setup to route port 80 and the others use port 80 below so we should remain consistent.  Same comment for two above.

Reply:
We're currently pointing to the origin nodes specifically because of HAP issues. When those are worked out I can switch them back.